src/app/components/form/input/FormInput.vue

```diff
@@ -51,6 +42,15 @@
             @focusout="value?.$touch()"
             @input="emit('input', ($event.target as HTMLInputElement)?.value)"
           />
+          <slot name="clearButton">
+            <ButtonIcon
+              :aria-label="t('iconAltClose')"
+              class="hidden flex-shrink-0 text-(--semantic-accent-accent-icon) peer-[:not([data-empty='true'])]:block"
+              @click="emit('input', '')"
+            >
+              <IVibetypeClose :aria-label="t('iconAltClose')" class="size-6" />
```

Comment:
```suggestion
              <AppIconClose />
```

saves us from setting the label and class all the time

Reply:
Replaced, you can resolve once seen.